Dockerfile.redhat

```diff
@@ -316,8 +316,8 @@ ARG minitrace_flags
 COPY ci/check_coverage.bat /ovms/
 ARG CHECK_COVERAGE=0
 ARG RUN_TESTS=0
-COPY rununittest.sh prepare_llm_models.sh /ovms/
-RUN if [ "$RUN_TESTS" == "1" ] ; then ./prepare_llm_models.sh /ovms/src/test/llm_testing docker && ./rununittest.sh ; fi
+COPY run_unit_tests.sh prepare_llm_models.sh prepare_gpu_models.sh /ovms/
+RUN if [ "$RUN_TESTS" == "1" ] ; then ./prepare_llm_models.sh /ovms/src/test/llm_testing docker && ./run_unit_tests.sh ; fi
```

Comment:
can we make test names distinctive for GPU? The easiest way would be to add words like "OpenCL", "VAContext" or "GPU" to each test name. It we want to check if GPU tests were run it would make the validation much easier

Reply:
You can already start using those names OpenCL and VAContext